pkg/gateway/model_build_targetgroup.go

```diff
@@ -341,6 +355,15 @@ func (t *latticeServiceModelBuildTask) buildTargetGroupSpec(ctx context.Context,
 				return latticemodel.TargetGroupSpec{}, err
 			}
 		}
+
+		var err error
```

Comment:
nit: you can remove it

Reply:
The reason i can't remove is because
```
ipAddressType, err := ...
```

will make `ipAddressType` a new variable inside the scope of the else statement instead of the one outside